quickwit-proto/src/quickwit.rs

```diff
@@ -261,389 +261,4 @@ pub enum OutputFormat {
 > ; fn call (& mut self , request : tonic :: Request < super :: FetchDocsRequest >) -> Self :: Future { let inner = self . 0 . clone () ; let fut = async move { (* inner) . fetch_docs (request) . await } ; Box :: pin (fut) } } let accept_compression_encodings = self . accept_compression_encodings ; let send_compression_encodings = self . send_compression_encodings ; let inner = self . inner . clone () ; let fut = async move { let inner = inner . 0 ; let method = FetchDocsSvc (inner) ; let codec = tonic :: codec :: ProstCodec :: default () ; let mut grpc = tonic :: server :: Grpc :: new (codec) . apply_compression_config (accept_compression_encodings , send_compression_encodings) ; let res = grpc . unary (method , req) . await ; Ok (res) } ; Box :: pin (fut) } "/quickwit.SearchService/LeafSearchStream" => { # [allow (non_camel_case_types)] struct LeafSearchStreamSvc < T : SearchService > (pub Arc < T >) ; impl < T : SearchService > tonic :: server :: ServerStreamingService < super :: LeafSearchStreamRequest > for LeafSearchStreamSvc < T > { type Response = super :: LeafSearchStreamResult ; type ResponseStream = T :: LeafSearchStreamStream ; type Future = BoxFuture < tonic :: Response < Self :: ResponseStream > , tonic :: Status > ; fn call (& mut self , request : tonic :: Request < super :: LeafSearchStreamRequest >) -> Self :: Future { let inner = self . 0 . clone () ; let fut = async move { (* inner) . leaf_search_stream (request) . await } ; Box :: pin (fut) } } let accept_compression_encodings = self . accept_compression_encodings ; let send_compression_encodings = self . send_compression_encodings ; let inner = self . inner . clone () ; let fut = async move { let inner = inner . 0 ; let method = LeafSearchStreamSvc (inner) ; let codec = tonic :: codec :: ProstCodec :: default () ; let mut grpc = tonic :: server :: Grpc :: new (codec) . apply_compression_config (accept_compression_encodings , send_compression_encodings) ; let res = grpc . server_streaming (method , req) . await ; Ok (res) } ; Box :: pin (fut) } _ => Box :: pin (async move { Ok (http :: Response :: builder () . status (200) . header ("grpc-status" , "12") . header ("content-type" , "application/grpc") . body (empty_body ()) . unwrap ()) }) , } } } impl < T : SearchService > Clone for SearchServiceServer < T > { fn clone (& self) -> Self { let inner = self . inner . clone () ; Self { inner , accept_compression_encodings : self . accept_compression_encodings , send_compression_encodings : self . send_compression_encodings , } } } impl < T : SearchService > Clone for _Inner < T > { fn clone (& self) -> Self { Self (self . 0 . clone ()) } } impl < T : std :: fmt :: Debug > std :: fmt :: Debug for _Inner < T > { fn fmt (& self , f : & mut std :: fmt :: Formatter < '_ >) -> std :: fmt :: Result { write ! (f , "{:?}" , self . 0) } } impl < T : SearchService > tonic :: transport :: NamedService for SearchServiceServer < T > { const NAME : & 'static str = "quickwit.SearchService" ; } }
```

Comment:
what is this format? This is annoying to commit such a change? do you know where this format change is coming from?

Reply:
It's generated code, I don't know why it generates the code differently in my case

Reply:
Can you remove from the PR the commits with auto generated code? We can't impact everyone on this change if we did not understand why it is happening.

Reply:
yep restored